test/Libraries/DynamoPythonTests/PythonEvalTests.cs

```diff
@@ -195,5 +195,34 @@ class myobj:
                 Assert.AreEqual("Output could not be converted to a .NET value", exc.Message);
             }
         }
+
+        [Test]
+        public void NonListIterablesCanBeOutput()
```

Comment:
does this test hit the condition on line `204`?

Reply:
Not this test. One which failed without that part is this one https://github.com/DynamoDS/Dynamo/blob/master/test/Libraries/DynamoPythonTests/PythonEvalTestsWithLibraries.cs#L189

Reply:
so it's currently failing?

Reply:
No, sorry if I didn't say it right. That test does work and hits that line.

What I meant is that at some point during my development of this I noticed the test failed, which triggered the changes now in line 204.